presentation/Main.java

```diff
@@ -30,13 +32,25 @@ public static void main(String[] args) {
                     System.out.print("Last Name: ");
                     studentToAdd.setLastName(scanner.next());
                     System.out.println();
-                    studentService.addStudent(studentToAdd);
+                    try {
+                        studentService.addStudent(studentToAdd);
+                    } catch (StudentIdAlreadyExistsException e) {
+                        System.err.println(e.getMessage());
+                        break;
+                    }
                     System.out.println();
                 }
                 case Menu.UPDATE_STUDENT -> {
                     System.out.print("Enter the StudentId to update: ");
                     String studentId = scanner.next();
-                    Student student = studentService.getStudentById(studentId);
+                    Student student;
+                    try{
+                        student = studentService.getStudentById(studentId);
+                    }
+                    catch (NoSuchStudentIdExistsException e){
+                        System.err.println(e.getMessage());
+                        break;
```

Comment:
Does this break really end the execution of the case `Menu.UPDATE_STUDENT` block?